src/orquestra/quantum/testing/mocks.py

```diff
@@ -1,23 +1,20 @@
 ################################################################################
 # © Copyright 2020-2022 Zapata Computing Inc.
 ################################################################################
-from ..api.backend import QuantumBackend
-from ..backends import SymbolicSimulator
-from ..circuits import Circuit
-from ..measurements import Measurements
+from orquestra.quantum.circuits import Circuit
+from orquestra.quantum.measurements import Measurements
+from orquestra.quantum.runners.symbolic_simulator import SymbolicSimulator
 
 
-class MockQuantumBackend(QuantumBackend):
+class MockCircuitRunner:
 
     supports_batching = False
 
     def __init__(self):
         super().__init__()
         self._simulator = SymbolicSimulator()
 
-    def run_circuit_and_measure(
+    def run_and_measure(
         self, circuit: Circuit, n_samples: int, **kwargs
     ) -> Measurements:
-        super(MockQuantumBackend, self).run_circuit_and_measure(circuit, n_samples)
-
-        return self._simulator.run_circuit_and_measure(circuit, n_samples)
+        return self._simulator.run_and_measure(circuit, n_samples)
```

Comment:
Include 

```python
    def get_measurement_outcome_distribution(
        self, circuit: Circuit, n_samples: Optional[int]
    ) -> MeasurementOutcomeDistribution:
        measurements = self._simulator.run_and_measure(circuit, n_samples)
        return measurements.get_distribution()

```

Reply:
Implemented in [934e0ba](https://github.com/zapatacomputing/orquestra-quantum/pull/66/commits/934e0baf30fef68d3051fa380fc0cc0d5f5f6d10)